cypress/e2e/spec.cy.js

```diff
@@ -0,0 +1,5 @@
+describe('My First Test', () => {
+  it('Visits our Application', () => {
+    cy.visit('http://localhost:19006/')
```

Comment:
Write a simple test:
```suggestion
    cy.visit('http://localhost:19006/')
    cy.contains('Register').should('be.visible') // Assert that element is visible

```